Scripts/Input/OVRTouchInputToEvents.cs

```diff
@@ -1,164 +1,167 @@
-﻿using System;
-using UnityEngine;
-using UnityEngine.InputNew;
-using UnityEngine.Experimental.EditorVR;
+﻿#if UNITY_EDITOR
+using System;
+using UnityEditor.Experimental.EditorVR;
 
 [assembly: OptionalDependency("OVRInput", "ENABLE_OVR_INPUT")]
 
-/// <summary>
-/// Sends events to the input system based on native Oculus SDK calls
-/// </summary>
-public class OVRTouchInputToEvents : MonoBehaviour
+namespace UnityEngine.InputNew
```

Comment:
I left it in the default namespace that the input-prototype uses. Originally, this was in the input-prototype repo and we pulled it out at one point. All of this code will get replaced soon with the new input system, including the whole UnityEngine.InputNew namespace.